scripts/firm.py

```diff
@@ -3,15 +3,17 @@
 def get_r(L, K, alpha, A, delta):
     '''
     The interest rate implied by the firm FOC for the choice of capital
+    Referring to equation (4.13)
     '''
     r = alpha * A * (L / K) ** (1 - alpha) - delta
 
     return r
 
 
-def get_w(r, alpha, A, delta):
+def get_w(L, K, alpha, A, delta):
```

Comment:
I think you are doing to want to keep a `get_w` function that returns the wage rate that corresponds to a given interest rate.  This might differ from the text, but that's ok.

Reply:
I am referring to equations 4.13 and 4.14. Should I keep the same function as what you did in class to make it simpler then?

Reply:
4.13 and 4.14 are what we used in class - rearranging of some terms gives us what was used in the code that Jason used. 8.16 and 8.17 also simplify to the same thing if one gets rid of labor augmenting technological change.

Reply:
That makes sense. The arguments included r previously but not the function. The function was defined based on L and K. So let's change all function determined by r. And I noticed get_w is used later but I do not remember if the arguments are defined right.

Reply:
I've got a pull request up right now that has `firm.py` the way we want it to be (at least for the time being).

Reply:
Ok. So I do not change it again because it causes conflicts. I will check other functions for its arguments now